controllers/admin/AdminImportController.php

```diff
@@ -1635,6 +1635,16 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
 
         $product = new Product($id_product);
 
+        if(!$product->id && (!isset($info['name']) || empty($info['name']))) {
+            $this->errors[] = sprintf(
+                $this->trans('Product with reference %1$s (ID: %2$s) cannot be saved : product name missing', [], 'Admin.Advparameters.Notification'),
+                (isset($info['reference']) && !empty($info['reference'])) ? Tools::htmlentitiesUTF8($info['reference']) : 'null',
+                !empty($info['id']) ? Tools::htmlentitiesUTF8($info['id']) : 'null'
+            );
+
+            return;
+        }
```

Comment:
```suggestion
        if (!$product->id && empty($info['name'])) {
            $this->errors[] = sprintf(
                $this->trans('Product with reference %1$s (ID: %2$s) cannot be saved: product name is missing', [], 'Admin.Advparameters.Notification'),
                (!empty($info['reference'])) ? Tools::htmlentitiesUTF8($info['reference']) : 'null',
                !empty($info['id']) ? Tools::htmlentitiesUTF8($info['id']) : 'null'
            );

            return;
        }
```

No need to check `isset` if you use `empty` 👍🏻 Also, a little change around wording as there's no need to use spaces before `:`.